FirebaseAuthUI/FIRAuthUIBaseViewController.h

```diff
@@ -104,6 +104,12 @@ typedef void (^FIRAuthUIAlertActionHandler)(void);
  */
 - (void)pushViewController:(UIViewController *)viewController;
 
+/** @fn onBackButtonPressed:
+    @brief Pops the view controller from navigation stack. If current controller is root 
+    works as @c cancelAuthorization
+ */
+- (void)onBackButtonPressed;
```

Comment:
Consider following the UIKit pattern of having a sender and passing the button in along with the event.

```objc
- (void)onBackButtonPressed:(id)sender;
```